src/longopt/grdmask_inc.h

```diff
@@ -25,30 +25,21 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
+	{ 0, 'A', "straightlines|steps",
+	          "m,p,x,y",           "meridian,parallel,x,y",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
-	          "",                  "",
+	{ 0, 'C', "clobber",
+	          "f,l,o,u",           "first,low,last,high",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'S', "",
-	          "",                  "",
+	GMT_G_OUTGRID_KW,
+	GMT_I_INCREMENT_KW,
+	{ 0, 'N', "outedgein",
```

Comment:
In GMT.jl this is `out_edge_in`

Reply:
OK, s that should definitively be an alias.  We can handle **outedgein**|**out_edge_in** I think.